javascript/javascript1/week1/housey-pricey.js

```diff
@@ -0,0 +1,29 @@
+// peter's house description
+let width = 8;
+let depth = 10;
+let height = 10;
+let gardenSizeInM2 = 100;
+let volumeInMeters = width * depth * height;
+let housePrice = (volumeInMeters * 2.5) * 1000 + (gardenSizeInM2 * 300);
+
+if (housePrice > 2500000) {
+    console.log(" Peter Paying more price: he has to pay :" + housePrice + "$");
+} else {
+    console.log(" Peter Paying less price: he is paying :" + housePrice + "$");
+
+}
+
+// Julia's house description
+width = 5;
+depth = 11;
+height = 8;
+gardenSizeInM2 = 70;
+volumeInMeters = width * depth * height;
+housePrice = (volumeInMeters * 2.5) * 1000 + (gardenSizeInM2 * 300);
+
+if (housePrice > 1000000) {
+    console.log(" Julia Paying more price: she has to pay :" + housePrice + "$");
+} else {
+    console.log(" Julia Paying less price: she is paying :" + housePrice + "$");
+
+}
```

Comment:
you can do this using arrays instead of re-assigning. like this 
const width = [8,5],
height=[10,11],
depth=[10,8]; 
same way for volume and houseprice.
you can compare using their index values (just suggestion.!!)